packages/host/app/services/matrix-service.ts

```diff
@@ -54,10 +55,21 @@ export type OperatorModeContext = {
   openCards: CardDef[];
 };
 
+export type AuthMode = 'login' | 'register' | 'forgot-password';
+
+export type AuthState = {
```

Comment:
I am wondering if this data should be a concern of this service. `AuthMode` sounds more of a navigational issue (login, register, forgot password) and the rest of the data in this type sounds like it is only temporary params for the password reset flow. I think it's better if we only have long lived state here. Perhaps it would be clearer if we had an auth mode service so the password reset params are separated from the matrix service state?

Reply:
Updated. [9c17253](https://github.com/cardstack/boxel/pull/921/commits/9c1725319658cbd1733310e2e79500bf9cf0a988)